### PR TITLE
SEO-174654-ASP-MVC-Controls-ej2-UG

### PR DESCRIPTION
**Pull request by Omanyadickson**
Hi @MallikaRK 

Kindly review my PR below. Thank you.

Regards.
Dickson J

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Here, the theme and script is referred using CDN inside the `<head>` of `~/Pages
 {% endhighlight %}
 {% endtabs %}
 
-N> Checkout the [Themes topic](https://ej2.syncfusion.com/aspnetcore/documentation/appearance/theme/) to learn different ways ([CDN](https://ej2.syncfusion.com/aspnetcore/documentation/common/adding-script-references#cdn-reference), [NPM package](https://ej2.syncfusion.com/aspnetcore/documentation/common/adding-script-references#node-package-manager-npm), and [CRG](https://ej2.syncfusion.com/aspnetcore/documentation/common/custom-resource-generator/)) to refer styles in ASP.NET Core application, and to have the expected appearance for Syncfusion ASP.NET Core controls.
+N> Checkout the [Themes topic](https://ej2.syncfusion.com/aspnetcore/documentation/appearance/theme) to learn different ways ([CDN](https://ej2.syncfusion.com/aspnetcore/documentation/common/adding-script-references#cdn-reference), [NPM package](https://ej2.syncfusion.com/aspnetcore/documentation/common/adding-script-references#node-package-manager-npm), and [CRG](https://ej2.syncfusion.com/aspnetcore/documentation/common/custom-resource-generator/)) to refer styles in ASP.NET Core application, and to have the expected appearance for Syncfusion ASP.NET Core controls.
```
Comment (Yvone-Atieno) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md:74]: Check CRG link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md (hunk 1 above)
Comment (Omanyadickson) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/getting-started-core.md:74]: Checked

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**